iceoryx_examples/iceperf/README.md

```diff
@@ -164,3 +167,125 @@ Wich technologies are measured depends on the operating system (e.g. no message
     Waiting for subscriber to unsubscribe ... 
     Finished!
 
+## Code walkthrough
+
+Here we roughly describe the setup for performing the measurements. Things like initialization, sending and receiving of data are technology specific and can be found in the respective files (e.g. uds.cpp for 
+unix doamin socket). Our focus here is on the abstraction layer on top which allows us to add new IPC tecehnologies or you to extend and compare with whatever.
+
+### iceperf-laurel application
+
+Besides includes for the different IPC technologies, the topic_data.hpp file is included that contains the PerTopic struct which is used to transfer some information between the applications. Independent of the real payload size, this struct is used as some kind of header in each transferred sample. 
+
+```cpp
+    struct PerfTopic
+    {
+        uint32_t payloadSize{0};
+        uint32_t subPackets{0};
+        bool run{true};
+    };
+```
+
+With `payloadSize` as the payload size used for the current measurement, `subPackets` as the number of sub-packets the payload is divided into when it is not possible to transfer it with one packet and `run` to shutdown iceperf-hardy at the end of the benchmark.
+
+Let's set some constants to prevent magic values. The default number of round trips is set and names for the communication resources that are used. 
+```cpp
+    constexpr int64_t NUMBER_OF_ROUNDTRIPS{10000};
```

Comment:
Is this value the "100000" from above table? Maybe use same value...

Reply:
This is the default value. In the example output we increased the value (via the command line parameter) to have more accurate results